discovery/targetgroup/targetgroup_test.go

```diff
@@ -40,14 +39,6 @@ func TestTargetGroupStrictJsonUnmarshal(t *testing.T) {
 				{"__address__": "localhost:9090"},
 				{"__address__": "localhost:9091"}}, Labels: model.LabelSet{"my": "label"}},
 		},
-		{
-			json: `	{"label": {},"targets": []}`,
-			expectedReply: errors.New("json: unknown field \"label\""),
-		},
```

Comment:
does that really break? I had the feeling that json unmarshal was strict

Reply:
It breaks, yes. `json.Unmarshal()` ignores extra fields.